Add note on index backfill (#5066)
yugabyte/yugabyte-db: docs/content/latest/api/ysql/commands/ddl_create_index.md

`CONCURRENTLY`, `USING method`, `COLLATE`, and `TABLESPACE` options are not yet supported.

{{< note title="Note" >}}

> When an index is created on an existing table, YugabyteDB will automatically backfill existing data
> into the index. However, currently, this is not done in an online manner. To online backfill an
> index, you can set `ysql_disable_index_backfill` gflag to `false` while starting yb-tservers. Note that we don't recommend setting this
> gflag in a production cluster yet. For more information on how online index backfill works,
> please see [Online Index Backfill](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/online-index-backfill.md)

Reviewer: ```suggestion
When an index is created on an existing table, YugabyteDB automatically backfills existing data
into the index. Currently, this is not done in an online manner. To online backfill an
index, you can set the `ysql_disable_index_backfill` flag to `false` when starting yb-tservers. Note that we don't recommend setting this flag in a production cluster yet. For details on how online index backfill works, see [Online Index Backfill](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/online-index-backfill.md)
```
